zipline/algorithm.py

```diff
@@ -1226,6 +1231,18 @@ def order(self,
         order_id : str
             The unique identifier for this order.
 
+        Notes
+        -----
+        The ``limit_price`` and ``stop_price`` arguments are mutually exclusive
+        with the ``style`` argument. ``limit_price`` and ``stop_price``
+        convenient shorthands for ``LimitOrder``, ``StopOrder``, and
+        ``StopLimitOrder`` execution styles. For example: ``limit_price=n`` is
+        equivalent to and mutually exclusive with
```

Comment:
Using the phrase "mutually exclusive" twice  in consecutive sentences reads awkwardly.  How about this?

```
The ``limit_price`` and ``stop_price`` arguments provide shorthands for passing common execution styles.  Passing ``limit_price=N`` is equivalent to ``style=LimitOrder(N)``.  Similarly, passing ``stop_price=M`` is equivalent to ``style=StopOrder(M)``, and passing ``limit_price=N`` and ``stop_price=M`` is equivalent to ``style=StopLimitOrder(N, M)``.  It is an error to pass both a style and ``limit_price`` or ``stop_price``.
```